src/pressure.rs

```diff
@@ -134,11 +135,57 @@ mod test {
 
     #[test]
     fn test_parse_pressure_record_errs() {
-        assert!(parse_pressure_record("avg10=2.10 avg60=0.12 avg300=0.00 total=391926")
-                .is_err());
-        assert!(parse_pressure_record("some avg10=2.10 avg300=0.00 total=391926")
-                .is_err());
-        assert!(parse_pressure_record("some avg10=2.10 avg60=0.00 avg300=0.00")
-                .is_err());
+        assert!(parse_pressure_record("avg10=2.10 avg60=0.12 avg300=0.00 total=391926").is_err());
+        assert!(parse_pressure_record("some avg10=2.10 avg300=0.00 total=391926").is_err());
+        assert!(parse_pressure_record("some avg10=2.10 avg60=0.00 avg300=0.00").is_err());
+    }
+
+    #[test]
+    fn test_mem_pressure() {
+        if !Path::new("/proc/pressure/memory").exists() {
+            return;
+        }
+
+        let mem_psi = memory_pressure().unwrap();
+        assert!(valid_percentage(mem_psi.some.avg10));
+        assert!(valid_percentage(mem_psi.some.avg60));
+        assert!(valid_percentage(mem_psi.some.avg300));
+        assert!(mem_psi.some.total > 0);
+
+        assert!(valid_percentage(mem_psi.full.avg10));
+        assert!(valid_percentage(mem_psi.full.avg60));
+        assert!(valid_percentage(mem_psi.full.avg300));
+        assert!(mem_psi.full.total > 0);
```

Comment:
On my test system, this assertion fails because total == 0:

```
some avg10=0.00 avg60=0.00 avg300=0.00 total=0
full avg10=0.00 avg60=0.00 avg300=0.00 total=0
```

Reply:
removed the assertion (it's meaningless to check u64 >= 0).